src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,160 @@
+package duke;
+
+import duke.task.Task;
+import duke.task.TaskList;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+
+/** Deals with user interactions. */
+public class Ui {
+    public static final String HORIZONTAL_BAR =
+            "____________________________________________________________";
+    public static final String NEWLINE = System.lineSeparator();
+    public static final String INDENTATION = "    ";
```

Comment:
I like that you declared these format constants outside so it's easier to change the Ui look next time.